modules/nlu/src/views/lite/intentEditor/style.scss

```diff
@@ -0,0 +1,295 @@
+.intentEditor {
+  display: flex;
+  justify-content: flex-end;
+  flex-grow: 1;
+  > div:first-child {
+    flex-grow: 1;
+    padding: 10px 25px;
+  }
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  margin: 20px 0 10px 0;
+
+  .hint {
+    color: #738694;
+  }
+}
+
+.editor-body {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: stretch;
+}
+
+.utterances {
+  flex: 1;
+  min-width: 300px;
+}
+
+.entities-list {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: flex-start;
+}
+
+p.utterance {
+  margin: 0;
+  font-size: 16px;
+  line-height: 25px;
+  z-index: 0;
+  display: flex;
+
+  &.title {
+    font-weight: bold;
+    line-height: 40px;
+    font-size: 25px;
+  }
+  &.active {
+    background-color: #eee;
+  }
+  &.wrong {
+    color: red;
+  }
+
+  .index {
+    width: 20px;
+    margin-right: 5px;
+    opacity: 0.5;
+    text-align: center;
+
+    &.count {
+      width: inherit;
+      background: lightgrey;
+      padding: 0 5px;
+      border-radius: 5px;
+    }
+  }
+}
+
+.slotMenu {
+  position: absolute;
+  display: block;
+  z-index: 1000;
+  background-color: #fefefe;
+  padding: 10px;
+  border: solid 1px #ccc;
+  border-radius: 5px;
+  box-shadow: 0px 3px 8px -2px rgba(0, 0, 0, 0.2);
+  p:first-child {
+    font-weight: bold;
+  }
+}
+
+.slotMenuItem {
+  margin: 0 5px 5px 0;
+}
+
+.slotMark {
+  cursor: pointer;
+  &:hover {
+    font-weight: bold;
+  }
+}
+
+/* ---------- */
+/* SLOTS COLORING*/
+/* ---------- */
+
```

Comment:
Oh noes, what have you done with the "tear your own eyes out" vintage slots coloring ???